EventListener/HttpCacheListener.php

```diff
@@ -114,6 +114,16 @@ public function onKernelResponse(FilterResponseEvent $event)
             $response->setMaxAge($age);
         }
 
+        if (!$response->headers->hasCacheControlDirective('max-stale') && null !== $stale = $configuration->getMaxStale()) {
+            if (!is_numeric($stale)) {
+                $now = microtime(true);
+
+                $stale = ceil(strtotime($configuration->getMaxStale(), $now) - $now);
```

Comment:
Could use `$stale` here instead of `$configuration->getMaxStale()`

Reply:
I changed it.